students/Kristy_Martini/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,134 @@
+import math
+
+def exchange_first_last(seq):
+    if (type(seq) is int or type(seq) is str):
+        was_int = False
+        if (type(seq) is int):
+            seq = str(seq)
+            was_int = True
+        new_seq = seq[len(seq)-1] + seq[1:len(seq)-1] + seq[0]
+        if was_int:
+            new_seq = int(new_seq)
+    else: 
+        was_tuple = False
+        if(type(seq) is tuple):
+            seq = list(seq)
+            was_tuple = True
+        first_element = seq[0]
+        last_element = seq[len(seq)-1]
+        seq[0] = last_element
+        seq[len(seq)- 1] = first_element
+        if was_tuple:
+            new_seq = tuple(seq)
+    print(new_seq)
+    return new_seq
+
+def remove_every_other(seq):
+    i = 0
+    if(type(seq) is int or type(seq) is str):
+        was_int = False
+        if (type(seq) is int):
+            seq = str(seq)
+            was_int = True
+        new_seq = ''
+        while i < len(seq):
+            new_seq += seq[i]
+            i += 2
+        if was_int:
+            new_seq = int(new_seq)
+    else: 
+        was_tuple = False
+        if(type(seq) is tuple):
+            seq = list(seq)
+            was_tuple = True
+        new_seq = list()
+        while i < len(seq):
+            new_seq.append(seq[i])
+            i += 2
+        if was_tuple:
+            new_seq = tuple(new_seq)
+    print(new_seq)
+    return new_seq
```

Comment:
Each one of these should be done using slicing.  Might take another look at this one.